src/sentry/auth/helper.py

```diff
@@ -233,16 +233,18 @@ def _handle_new_membership(self, auth_identity: ApiAuthIdentity) -> ApiOrganizat
         )
 
         if om is not None:
-            audit_log_service.log_organization_membership(
-                metadata=AuditLogMetadata(
-                    organization=self.organization,
-                    actor=user,
+            log_service.record_audit_log(
+                event=AuditLogEvent(
+                    organization_id=self.organization.id,
+                    time_of_creation=timezone.now(),
```

Comment:
Should we stick closer to the conventions we use in the schema? `AuditLogEntry` uses `datetime` but many of our other models use `date_added` for a 'created' timestamp.

Reply:
Sure.  I have this addressed in a follow up PR.